src/monitoring_service/handlers.py

```diff
@@ -370,6 +370,10 @@ def updated_head_block_event_handler(event: Event, context: Context) -> None:
 
 
 def _is_mr_valid(monitor_request: MonitorRequest, channel: Channel) -> bool:
+    if monitor_request.non_closing_signer != monitor_request.non_closing_participant:
```

Comment:
Do we have a test for this?

Reply:
I'll add one and move this to the request collector.